es.hjson

```diff
@@ -510,65 +510,65 @@
 
-  "marry.marrystats.CalsiBot": "<b>El diferencia de usted, CalsiBot es puro y no tiene antecedentes matrimoniales.</b>"
+  "marry.marrystats.CalsiBot": "<b>A diferencia de usted, CalsiBot es puro y no tiene antecedentes matrimoniales.</b>"
 
-  "marry.marrystats.own.information": "💖 %s<b> - Casar Estadísticas - 1️⃣:</b>\n\n🙂 - Felicidad: <b>%s</b>\n🥺 - Niños: <b>%s</b>\n🏠 - Casas: <b>%s</b>\n💰 - Patrimonio neto: <b>$%s</b>"
-  "marry.marrystats.own.marriages": "💖 %s<b> - Casar Estadísticas - 2️⃣:</b>\n\n❤️ <b>Matrimonios:</b>\nSolicitudes más aceptadas por %s\nSolicitudes rechazadas más por %s\nCasado: <b>%s</b> El | Rechazado: <b>%s</b>"
-  "marry.marrystats.own.divorces": "💖 %s<b> - Casar Estadísticas - 3️⃣:</b>\n\n🖤 <b>Divorcios:</b>\nDivorciado de %s más\nTotal: <b>%s</b>"
+  "marry.marrystats.own.information": "💖 %s<b> - Estadísticas de matrimonio - 1️⃣:</b>\n\n🙂 - Felicidad: <b>%s</b>\n🥺 - Niños: <b>%s</b>\n🏠 - Casas: <b>%s</b>\n💰 - Patrimonio neto: <b>$%s</b>"
+  "marry.marrystats.own.marriages": "💖 %s<b> - Estadísticas de matrimonio - 2️⃣:</b>\n\n❤️ <b>Matrimonios:</b>\nSolicitudes más aceptadas por %s\nSolicitudes más rechazadas por %s\nCasado: <b>%s</b> | Rechazado: <b>%s</b>"
+  "marry.marrystats.own.divorces": "💖 %s<b> - Estadísticas de matrimonio - 3️⃣:</b>\n\n🖤 <b>Divorcios:</b>\nDivorciado más de %s\nTotal: <b>%s</b>"
 
 
   /* ---------------------------------------------------------------------
                     Permaban - Used for /Permaban, /UnPermaban
      --------------------------------------------------------------------- */
-  "permaban.permaban": "Permaban"
+  "permaban.permaban": "Ban permanente"
 
   "permaban.callbacks": {
-    "userPermabanned": "<a href=\"tg://user?id=%s\">Este usuario</a><b> fue Permabanned por </b>%s"
-    "userPermabannedAnswerText": "Usuario permanente"
-    "userAlreadyPermabanned": "<a href=\"tg://user?id=%s\">Este usuario</a><b> ya está Permabanned.</b>"
-    "userAlreadyPermabannedAnswerText": "Usuario ya permabanneado"
+    "userPermabanned": "<a href=\"tg://user?id=%s\">Este usuario</a><b> fue baneado permanentemente por </b>%s"
+    "userPermabannedAnswerText": "Usuario baneado permanente"
+    "userAlreadyPermabanned": "<a href=\"tg://user?id=%s\">Este usuario</a><b> ya está baneado permanentemente.</b>"
+    "userAlreadyPermabannedAnswerText": "Usuario ya baneado permanentemente"
 
-    "userUnPermabanned": "<a href=\"tg://user?id=%s\">Este usuario</a><b> fue no-Permabanned por </b>%s"
-    "userUnPermabannedAnswerText": "Usuario no permanente"
-    "userAlreadyNotPermabanned": "<a href=\"tg://user?id=%s\">Este usuario</a><b> ya no está Permabanned</b>"
-    "userAlreadyNotPermabannedAnswerText": "El usuario ya no está permabanneado"
+    "userUnPermabanned": "<a href=\"tg://user?id=%s\">Este usuario</a><b> fue desbaneado permanentemente por </b>%s"
+    "userUnPermabannedAnswerText": "Usuario desbaneado permanente"

```

Comment:
Is the meaning correct here?

It will be shown after a user has clicked a button, and should tell them that X user is now no longer permabanned.